src/main.cpp

```diff
@@ -6,12 +6,11 @@
 
 int main(int argc, char *argv[]) {
     // Create an instance of the DatabaseManager
-    DatabaseManager data_base("audio_station_loop.db", "database/schema/schema.sql");
+    DatabaseManager data_base("audio_station_loop.db", "/database/schema/schema.sql");

```

Comment:
This will create a directory at a low level. If we want to be more explicit, we it should have the ./database/.... to indicate it is on the same level as src